subgraphs/rocket-pool/src/entities/protocol.ts

```diff
@@ -0,0 +1,43 @@
+import { Protocol } from "../../generated/schema";
+import {
+  PROTOCOL_NAME,
+  PROTOCOL_SLUG,
+  PROTOCOL_SCHEMA_VERSION,
+  PROTOCOL_SUBGRAPH_VERSION,
+  PROTOCOL_METHODOLOGY_VERSION,
+  RETH_ADDRESS,
+  Network,
+  ProtocolType,
+  BIGDECIMAL_ZERO,
+} from "../utils/constants";
+
+export function getOrCreateProtocol(): Protocol {
+  let protocol = Protocol.load(RETH_ADDRESS);
+
+  if (!protocol) {
+    protocol = new Protocol(RETH_ADDRESS);
+
+    // Metadata
+    protocol.id = RETH_ADDRESS;
+    protocol.name = PROTOCOL_NAME;
+    protocol.slug = PROTOCOL_SLUG;
+    protocol.schemaVersion = PROTOCOL_SCHEMA_VERSION;
```

Comment:
Can you place the versions outside of the if block? The idea here is that if we need to graft it will still update the versions https://github.com/messari/subgraphs/blob/master/subgraphs/compound-forks/src/mapping.ts#L1883-L1885

Reply:
This is fixed now